### DISCUSSION TITLE
How do I hide an endpoint

### DISCUSSION THREAD

**Discussion by rabol** [dedoc/scramble#234] [Q&A]
Hi

how to I hide an endpoint so that it does not show up in the documentation?

**Comment by romalytvynenko** [comment]
@rabol you can provide your own route resolution function in which you can pick which routes are documented: https://scramble.dedoc.co/usage/getting-started#routes-resolution

**Comment by rabol** [comment]
Sure, but then one had to list all the routes manually

 @hideFromAPIDocumentation would be a nice feature

**Reply by romalytvynenko** [reply]
@rabol 

> had to list all the routes manually

I'm not sure I'm following. Can you elaborate what you mean here?

**Reply by makroxyz** [reply]
like scribe, @hideFromAPIDocumentation in action comment skip the documentation for the action

> @rabol
> 
> > had to list all the routes manually
> 
> I'm not sure I'm following. Can you elaborate what you mean here?

**Comment by makroxyz** [comment]
@rabol is there a workaround to skip also resource documentation?

**Comment by romalytvynenko** [comment]
@makroxyz @rabol 

https://scramble.dedoc.co/usage/getting-started#excluding-hiding-some-routes-from-api-documentation

Added `#[ExcludeRouteFromDocs]` attribute to exclude specific routes from documentation in `0.11.23`
